src/app/app.config.ts

```diff
@@ -0,0 +1,8 @@
+import { ApplicationConfig } from '@angular/core';
+import { provideRouter } from '@angular/router';
+
+import { routes } from './app.routes';
+
+export const appConfig: ApplicationConfig = {
+  providers: [provideRouter(routes)]
```

Comment:
suggestion

            providers: [provideRouter(routes)],

          Explantion: The use of provideRouter is appropriate, consider handling errors or cases where routes may not be defined properly.